qiita_db/test/test_analysis.py

```diff
@@ -95,12 +100,12 @@ def test_create(self):
 
         new = Analysis.create(User("admin@foo.bar"), "newAnalysis",
                               "A New Analysis")
-        self.assertEqual(new.id, 3)
+        self.assertEqual(new.id, 7)
```

Comment:
I don't think we need to make all changes at once. Since you're already modifying this test, please make the change so we don't have that burden on the future (it's just a small change and it will reduce any further change in the future). Thanks!